src/Pages/Footer/Footer.js

```diff
@@ -1,14 +1,47 @@
-import React from 'react'
-import styles from './Footer.module.css';
+import React from "react";
+import styles from "./Footer.module.css";
 
-Footer = () => {
+const Footer = () => {
   return (
-    <div>
-      
-    </div>
-  )
-}
-export default Footer
+    <footer>
+      <div className={styles.footerTop}>
+        <div>
+          <div className={styles.altSchool}>
+            <p>Alt School</p>
+            <p>
+              A software Engineering School-   Acquire in-demand tech skills
+              without hassle
+            </p>
+          </div>
+          <div className={styles.socials}>
+            <span>f</span>
+            <span>T</span>
+            <span>I</span>
+            <span>L</span>
+          </div>
+        </div>
+        <div className={styles.footerContactUs}>
+          <div>Contact Us</div>
```

Comment:
I was also lost in the footer section, cos I'm not sure this is traditional functional/arrow function component, but if it works.
Good work !

Probably some new thing I don't know hehe.
Nice work again, can't wait till you review my own work !